lib/tmuxinator/wemux_support.rb

```diff
@@ -5,17 +5,17 @@ def wemux?
     end
 
     def load_wemux_overrides
-      self.instance_eval do
-        def render
+      class_eval do
```

Comment:
Had to change this around due to the `Lint/NestedMethodDefinition` cop which didn't like the `def`s inside a method definition.  Added a test to make sure that the functionality stays consistent, but if could also just revert this and deactivate the cop itself.